lms/djangoapps/verify_student/urls.py

```diff
@@ -23,6 +23,16 @@
         }
     ),
 
+    # This is for A/B testing.
+    url(
+        r'^start-flow_ab/{course}/$'.format(course=settings.COURSE_ID_PATTERN),
```

Comment:
Have you considered using a query string parameter instead of adding a new URL? Instead of inspecting the request path, you could look for a querystring argument (e.g., alternate=1). It would require you to change more about the tests, though.